### ISSUE TITLE
Baselayer control doesn't select default baselayer if using more than one map

### ISSUE THREAD

**Issue by wowonk** [Leaflet/Leaflet#6618]
Hi,
here the example (leaflet V1.4.0):
https://aachen-hat-energie.de/test_ww/test_zukunft.htm
The baselayer-control is only activated on the third map, not on the first and second, as it should be. My Code:
>
    <link rel="stylesheet" href="../js/Leaflet/leaflet.css" >
    <script src="../js/Leaflet/leaflet.js" ></script>
    <script src="../js/Leaflet/leaflet-bing-layer.min.js"></script>
    <script>
	'use strict';
	var BING_KEY = "AplTrT4uzwlmfcERFFQu_NqDycERC_Er0qGYzZhIqrDfq-naYCsUr1kbbKRUqhq1";
	var osmUrl = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
    function show_karte(div_id, mittelpunkt, zoom){
	var bingLayer = L.tileLayer.bing({bingMapsKey: BING_KEY, maxZoom: 20, imagerySet:                  'Aerial', attribution: '&copy; 2019 Microsoft Corporation, Earthstar Geographics SIO'});
	var bingLabelLayer = L.tileLayer.bing({bingMapsKey: BING_KEY, maxZoom: 20, imagerySet: 'AerialWithLabels', attribution: '&copy; 2019 Microsoft Corporation, Earthstar Geographics SIO'});
	var	osmLayer = L.tileLayer(osmUrl, {maxZoom: 20, attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>'});
	var mittelpunkt = [mittelpunkt[1], mittelpunkt[0]]
	var map = L.map(div_id,{layers: [bingLabelLayer]}).setView(mittelpunkt, zoom);
	var baseLayers = {"Stra&szlig;enkarte": osmLayer, "Luftbild mit Stra&szlig;ennamen": bingLabelLayer,"Luftbild": bingLayer};	
	var control = L.control.layers(baseLayers).addTo(map);
	};	
    </script>
    </head>
     <body> 
    <div id="map" style="width:450px; height: 450px;">
    <script  type="text/javascript"> show_karte("map",[6.026173,50.816022],13); </script>
    </div>
    <div id="map1" style="width:550px; height: 450px;">
    <script  type="text/javascript"> show_karte("map1",[6.10428,50.76079],11); </script>
    </div>
    <div id="map2" style="width:650px; height:450px">  
    <script  type="text/javascript">show_karte("map2",[6.10428,50.76079],12); </script>
    </div>
    </body>

I tried solutions as 
https://aachen-hat-energie.de/test_ww/test_zukunft1.htm
https://aachen-hat-energie.de/test_ww/test_zukunft2.htm
which did not help. There seems to be only ONE control-object.
Sorry, I'm a dummy, but nobody on stackoverflow could help.

**Comment by IvanSanchez** [comment]
[Plunkr here](https://plnkr.co/edit/d8GxGerY0L7pX0qV7YBj?p=preview).

The word is not "activated", but rather "doesn't show any selected baselayer".

**Comment by IvanSanchez** [comment]
Can be due to https://github.com/Leaflet/Leaflet/commit/19d9e3bc93072bac655b9c6c28737442a6afe7fc . Should research whether several `<section>`s with radio buttons make exclusive groups of if they behave like one group of checkboxes.

**Comment by sc28** [comment]
I stumbled across this issue as well and thought I was doing something wrong, until I realized it might be a bug.
It seems their is no issue in Leaflet 0.7.2, while the issue is reproducible in Leaflet 1.4.0.

The following fiddles demonstrate the issue:
Leaflet 0.7.2 [http://jsfiddle.net/sc28/fvy8aq37/2/](http://jsfiddle.net/sc28/fvy8aq37/2/ ) --> the base layers are correctly selected in both map controls
Leaflet 1.4.0 [http://jsfiddle.net/sc28/9j3hstgx/](http://jsfiddle.net/sc28/9j3hstgx/) --> the base layers are only selected in the second map; when a base map is clicked in either map control, it gets deselected in the other.

**Comment by wowonk** [comment]
Hm, on my PC the second example works well too.

**Comment by sc28** [comment]
@wowonk Which browser are you using? I tried on Firefox and Chrome and the issue is reproduced in both. Which browser are you using?

**Comment by wowonk** [comment]
Hm, I tested on my tablett (MS Surface 3) with newest Fireforx, Win10, dirctX V12, Treiber WDDM 2.0: it works
I now tested on my PC with newest Firefox (Win8), dirctX V11, Treiber WDDM 1.3: I have the same errors as you
What now ???

**Comment by cherniavskii** [comment]
Duplicate of #6482. It's already fixed by #6483

**Comment by IvanSanchez** [comment]
Wow, I forgot I approved such a PR :innocent:

**Comment by cherniavskii** [comment]
Tested it locally, the issue is gone.

**Comment by cherniavskii** [comment]
BTW @IvanSanchez, I'm going to release new Leaflet version this week.
Are there PRs that need to merged before the release?
